README.md

```diff
@@ -366,16 +366,16 @@ copycat.phoneNumber('foo')
 // => '+208438699696662'
 ```
 ```js
-copycat.phoneNumber('foo', { prefixes: ['+3319900', '+3363998'], min: 1000, max: 9999 })
-// => '+33639987662'
+copycat.phoneNumber('foo', { prefixes: ['+3319900', '+3363998'], minLength: 11, maxLength: 11 })
+// => '+3363998462'
 ```
 
 **note** The strings _resemble_ phone numbers, but will not always be valid. For example, the country dialing code may not exist, or for a particular country, the number of digits may be incorrect. Please let us know if you need valid
 phone numbers, and feel free to contribute :)
 
 #### `options`
-- **`min=10000000000`:** Constrain generated values to be greater than or equal to `min` allow to control the minimum number of digits in the phone number
-- **`max=999999999999999`:** Constrain generated values to be less than or equal to `max` allow to control the maximum number of digits in the phone number
+- **`minLength=maxLength ?? 12`:** Constrain generated values have a length greater than or equal to `minLength`. If `minLength` is not specified but `maxLength` is, `minLength` is taken as the given `maxLength`. If neither `minLength` or `maxLength` are given, `minLength` defaults to `12`.
+- **`maxLength=minLength ?? 16`:** Constrain generated values to have a length less than or equal to `maxLength`. If `maxLength` is not specified but `minLength` is, `minLength` is taken as the given `maxLength`. If neither `minLength` or `maxLength` are given, `maxLength` defaults to `16`
```

Comment:
@avallete I've made a change: if one only provides `minLength` or `maxLength` (not both), we use the given number as both the upper and lower bound. e.g. `phoneNumber(input, { maxLength: 4 })` will return `4`.

It probably sounds strange on first blush. The reason is: I don't know if its clear enough to users that if they provide `phoneNumber(input, { maxLength:  4 })`, we'll still be returning phone numbers of length `12` (since `12` is the default min, and if min > max we take the min).

Put differently, if the user specifies only one of the bounds, I doubt they're wanting to use the default for the other bound.

What do you think?

Reply:
In that case I wonder if it might be worth to just make it explicit at the types level, that it's either a range or a static length, something like:

<img width="349" alt="Screenshot 2024-02-06 at 14 40 01" src="https://github.com/snaplet/copycat/assets/8771783/87a77b3d-7c17-4c0f-a7d3-312c48f9b241">

Reply:
Great idea! Will do!